common/traits/btr_species_traits_habitability.txt

```diff
@@ -5,7 +5,7 @@ trait_human_habitability_preference = {
 	sorting_priority = 30
 	ai_weight = { weight = 0 }
 	modifier = {
-		pc_continental_habitability =  0.7
+		pc_continental_habitability =  1.0
```

Comment:
Shouldn't this be left at 0.7 as the game gives a 30% bonus to your homeworld class automatically?